l10n_es_aeat/models/l10n_es_aeat_report_tax_mapping.py

```diff
@@ -0,0 +1,194 @@
+            "amount": amount,
+            "move_line_ids": [(6, 0, move_lines.ids)],
+        }
+
+    def _get_partner_domain(self):
+        return []
+
+    def _get_move_line_domain(self, date_start, date_end, map_line):
+        self.ensure_one()
+        taxes = self.get_taxes_from_templates(map_line.tax_ids)
+        move_line_domain = [
+            ("company_id", "child_of", self.company_id.id),
+            ("date", ">=", date_start),
+            ("date", "<=", date_end),
+            ("exclude_from_invoice_tab", "=", True),
+        ]
+        if map_line.move_type == "regular":
+            move_line_domain.append(
+                ("move_id.move_type", "in", ("receivable", "payable", "liquidity"))
+            )
+        elif map_line.move_type == "refund":
+            move_line_domain.append(
+                ("move_id.move_type", "in", ("receivable_refund", "payable_refund"))
+            )
+        if map_line.field_type == "base":
+            move_line_domain.append(("tax_ids", "in", taxes.ids))
+        elif map_line.field_type == "amount":
+            move_line_domain.append(("tax_line_id", "in", taxes.ids))
+        else:  # map_line.field_type == 'both'
+            move_line_domain += [
+                "|",
+                ("tax_line_id", "in", taxes.ids),
+                ("tax_ids", "in", taxes.ids),
+            ]
+        if map_line.sum_type == "debit":
+            move_line_domain.append(("debit", ">", 0))
+        elif map_line.sum_type == "credit":
+            move_line_domain.append(("credit", ">", 0))
+        if map_line.exigible_type == "yes":
+            move_line_domain.append(("tax_exigible", "=", True))
+        elif map_line.exigible_type == "no":
+            move_line_domain.append(("tax_exigible", "=", False))
+        move_line_domain += self._get_partner_domain()
+        return move_line_domain
+
+    def _get_tax_lines(self, date_start, date_end, map_line):
+        """Get the move lines for the codes and periods associated
+
+        :param date_start: Start date of the period
+        :param date_end: Stop date of the period
+        :param map_line: Mapping line record
+        :return: Move lines recordset that matches the criteria.
+        """
+        domain = self._get_move_line_domain(date_start, date_end, map_line)
+        return self.env["account.move.line"].search(domain)
+
+    @api.model
+    def _prepare_regularization_move_line(self, account_group):
+        return {
+            "name": account_group["account_id"][1],
+            "account_id": account_group["account_id"][0],
+            "debit": account_group["credit"],
+            "credit": account_group["debit"],
+        }
+
+    def _process_tax_line_regularization(self, tax_lines):
+        self.ensure_one()
+        groups = self.env["account.move.line"].read_group(
+            [
+                ("id", "in", tax_lines.move_line_ids.ids),
+                ("exclude_from_invoice_tab", "=", True),
+            ],
```

Comment:
¿El dominio no debería ser?:
[
            ("id", "in", tax_lines.move_line_ids.ids),
            ("parent_state", '=', 'posted'),
 ]

Se ha eliminado el:

- ("exclude_from_invoice_tab", "=", True),
Debido a que al realizar facturas/asientos/apuntes manuales este campo solamente pasa a True cuando se trata de líneas con Impuestos aplicados, por el resto de casos es False.

Se ha incluido el:

- ("parent_state", '=', 'posted')
Debido a que no deberían tenerse en cuenta aquellos apuntes donde el padre (el asiento) no está publicado aún, ya que puede dar a errores/problemas en los cálculos de los modelos.